boards/shields/semtech_sx1262mb2das/Kconfig.defconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022 Konstantinos Papadopoulos <kostas.papadopulos@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_SEMTECH_SX1262MB2DAS
+
+if LORA
+
+config SPI
+	default y
+
+endif # LORA
```

Comment:
Nit: prefer the use of `depends on FOO` instead of `if FOO`, when there's only one symbol impacted